addons/gd_/GD_.gd

```diff
@@ -2589,7 +2589,11 @@ static func to_array(thing):
 		for key in keyed_iterable(thing):
 			array.append(thing[key])
 	return array
-static func to_finite(a=0, b=0, c=0): not_implemented()
+static func to_finite(value): 
+	if value is float or value is int and !is_nan(value) and !value == 2147483647 :
+		return value
+	return 2147483647
```

Comment:
Super cool of you to pitch in 🚀

Im not too familiar with this so just got a few questions
This number seems to be the max of int 32, is there a specific reason why its that number? 
As opposed to 9223372036854775807 (int 64)?